outlines/welcome.md

```diff
@@ -0,0 +1,31 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework (and in many cases other frameworks)
+  2. Nothing here is *required* (i.e. you can use the platform in many other ways), but this is how we think apps are best developed. Take what you want from it but be aware that these are the community conventions
+  3. Hard APIs can be found at [docs] and community packages at [atmosphere].
+2. What is Meteor?
+  1. A set of JS client + server code for building web and mobile applications, built and curated by MDG and the Meteor community.
+  2. A build tool for building JS bundles for deployment
+3. Meteor principles
+  1. Data on the Wire. Meteor doesn't send HTML over the network. The server sends data and lets the client render it.
+  1. One Language. Meteor lets you write both the client and the server parts of your application in JavaScript.
+  1. Database Everywhere. You can use the same methods to access your database from the client or the server.
+  1. Latency Compensation. On the client, Meteor prefetches data and simulates models to make it look like server method calls return instantly.
+  1. Full Stack Reactivity. In Meteor, realtime is the default. All layers, from database to template, update themselves automatically when necessary.
+  1. Embrace the Ecosystem. Meteor is open source and integrates with existing open source tools and frameworks.
+  1. Simplicity Equals Productivity. The best way to make something seem simple is to have it actually be simple. Meteor's main functionality has clean, classically beautiful APIs.
```

Comment:
I don't think this is a principle that Meteor actually embodies this - _using_ Meteor can be simple, but there's extreme complexity under the hood.